packages/salesforcedx-utils-vscode/src/commands/channelService.ts

```diff
@@ -12,18 +12,18 @@ import { nls } from '../messages';
 
 export class ChannelService {
   private readonly channel: OutputChannel;
-  private static instance: ChannelService;
+  private static instances: { [key: string]: ChannelService } = {};
```

Comment:
@lcampos - You should probably see this.  Was this the intent?  Currently, when we try to output to the SOQL Extension channel it prints to the LWC Extension Channel because this is a singleton with the channel stored on it...and that extension sets the channel name first.

Reply:
This change fixes that and i verified we have output in SOQL Extension channel.

Reply:
Thanks for catching that. Please add a test to verify you get the correct channel when there's existing ones https://github.com/forcedotcom/salesforcedx-vscode/blob/develop/packages/salesforcedx-utils-vscode/test/unit/commands/channelService.test.ts